spec.md

```diff
@@ -722,7 +770,7 @@ the current value of the element, writes are not visible until after a positive
 edges of the register's clock port.
 
 The clock signal for a register must be of type `Clock`{.firrtl}.  The type of a
-register must be a passive type (see [@sec:passive-types]).
```

Comment:
Probably don't want to allow const memories either?

Reply:
That is a very interesting question I don't know the answer to.  If firrtl supported initialization, then yes, constant memories would make sense.

Reply:
Went ahead and defined const memories as ROMs and no write-ports allowed.  Punted on how you get the data into the ROM.